Godeps/Godeps.json

```diff
@@ -832,12 +832,12 @@
 		},
 		{
 			"ImportPath": "github.com/docker/distribution/digest",
-			"Comment": "v2.4.0-rc.1-38-gcd27f17",
+			"Comment": "v2.4.0-rc.1-38-gcd27f179",
```

Comment:
A lot of the comments got bumped. Not the actual versions, but it added one extra digit of the git hash on there. I can remove if needed